src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,10 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <ul>
+    {movies.map(movie => (
+      <MovieCard key={movie.imdbId} movie={movie} />
+    ))}
```

Comment:
<img width="440" alt="image" src="https://github.com/mate-academy/react_movies-list-js/assets/86789161/89a79eb7-cbfc-45c0-b8b3-5b724de3b797">

Reply:
You don't have block with such classname